KIPs/kip-15.md

```diff
@@ -0,0 +1,136 @@
+---
+kip: 15
+title: Additional LM on ETH & Polygon
+status: Active
+type: Spend KNC
+authors: Kyber Network team
+created: 2021-09-06
+---
+
+```
+executor: short timelock executor
+voting_power: current staked voting power
+execution: transfer 95,350 knc to 0xD69D7A1031e6E63a162414F9A77278757690C30E (kyber multisig to handle KNC bridging between ethereum and other chains)
+start_timestamp: 1630998000 (Tue, 7 Sep 2021 07:00:00 UTC)
+end_timestamp: 1631603227 (Tue, 14 Sep 2021 07:07:07 UTC)
```

Comment:
this is the start time of new epoch, please reduce it by few seconds.

Reply:
Sure

Reply:
Can use this contract `0xfbd73581af42b26295CF4B88B74A5AF71F209F8b` and call function `validateProposalCreation(startTime, endTime)` to validate if start & end times are valid to create a new proposal